client/my-sites/current-site/sidebar-banner/style.scss

```diff
@@ -30,46 +30,3 @@
 		}
 	}
 }
-
-.sidebar-banner-new {
```

Comment:
As an aside, do be careful with using names like `old` and `new`. If we end up with 2+ iterations, what do we call the next items? For naming ideas, we could be more descriptive about what is new about the banner, or give temporary names, with a TODO to cleanup after the test is run.

Reply:
Thanks! It was indeed intended to be renamed after the test, but I agree a comment with explaination would make it clearer